### PR TITLE
Add Nelkinda and e-zest coderetreats in Pune.

### PR DESCRIPTION
**Pull request by christianhujer**

### CODE REVIEW

File: _data/events/Pune-2.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "title": "GDCR17 e-zest, Pune",
+    "url": "http://www.e-zest.com/gdcr-2017",
+    "moderators": [
+        "@christianhujer",
+        "@nelkinda",
+        "@ezest"
```
Comment (rradczewski) [inline _data/events/Pune-2.json]: Please move the sponsors to the [`sponsors`](https://github.com/coderetreat/coderetreat.github.io/blob/master/events/event_schema.json#L61-L67) field

File: _data/events/Pune.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "title": "GDCR17 Nelkinda, Pune",
+    "url": "http://nelkinda.com/events/2017/11/18/Global-Day-of-Coderetreat/pune-nelkinda",
+    "moderators": [
+        "@SiddheshNikude",
+        "@nelkinda"
```
Comment (rradczewski) [inline _data/events/Pune.json]: Please move the sponsors to the [`sponsors`](https://github.com/coderetreat/coderetreat.github.io/blob/master/events/event_schema.json#L61-L67) field

**Comment by christianhujer** [comment]
Moved sponsors to the `sponsor` field as requested. Thanks for the review!

**Comment by rradczewski** [comment]
Thanks, looks good! Welcome to gdcr17 ☺️🎉

**Pull request merged**